config/docs-json.config.js

```diff
@@ -73,16 +74,17 @@ function getLevel(ratio) {
 }
 
 function getContrastCheck(value, path, allTokens) {
-  const format = path.replaceAll('.', '-').replace('&', 'and');
-  const withPrefix = 'telekom-' + format;
-  const currentToken = allTokens.find((el) => el.name === withPrefix);
+  const currentToken = allTokens.find((el) => isEqual(el.path,path.split('.')));
   let contrastRatio;
-  if (currentToken && currentToken.value) {
-    const firstHexValue = tinycolor(value).toHexString();
-    const secondHexValue = tinycolor(currentToken.value).toHexString();
-    contrastRatio = hex(firstHexValue, secondHexValue);
+  if (!currentToken) {
+    throw Error(`Couldn't find token in contrast check for ${path}`)
```

Comment:
I think it's not actually needed but I would add `new` here, (`throw new Error`)